Includes additional set-up instructions for windows (#182)
NUbots/NUbook: src/book/03-guides/02-tools/03-webots.mdx

   3. On the computer you will be running the Webots simulator on, open the `NUWebots` repo in a text editor. Open the file `webots/projects/samples/contests/robocup/controllers/referee/game.json` (you can use <kbd>Ctrl</kbd>+<kbd>p</kbd>, search for `game.json` and the first file should be the correct one).

>    4. Under `red: hosts:` add the IP address given by the following command for any computers you are running the robot code on. For any robots running on the same computer as the Webots simulator, do not add its own hostname/IP address.

Reviewer: Robots running on the same computer as the webots simulation do not need their hostname or IP address listed.

>    4. Under `red: hosts:` add the IP address given by the following command for any computers you are running the robot code on. For any robots running on the same computer as the Webots simulator, do not add its own hostname/IP address.

Reviewer: ```suggestion
   4. Under `red: hosts:` add the IP address given by the following command for any computers you are running the robot code on. Robots running on the same computer as the webots simulation do not need their hostname or IP address listed.
```
